source/language/classical.rst

```diff
@@ -427,6 +427,278 @@ Terminating the program early
 The statement ``end;`` immediately terminates the program, no matter what scope
 it is called from.
 
+
+The Switch statement
+--------------------
+
+An OpenQASM3 ``switch`` statement shall use the following keywords:
+    - ``switch``
+    - ``case``
+    - ``default``
+    - ``break``
+
+An OpenQASM3 ``switch`` statement shall be the following grammar:
+
+      - the ``switch`` keyword
+      - a right paren '(' literal
```

Comment:
Can we please use `(`, ie, `` markup for this and other similar instances below as this is consistent with the rest of the document?